src/modules/fw_pos_control_l1/fw_pos_control_l1_main.cpp

```diff
@@ -2301,7 +2305,9 @@ void FixedwingPositionControl::tecs_update_pitch_throttle(float alt_sp, float v_
 	// we're in transition
 	if (_vehicle_status.is_vtol && _vehicle_status.in_transition_mode) {
 		_was_in_transition = true;
-		_asp_after_transition = _ctrl_state.airspeed;
+		// set this to transition airspeed to init tecs correctly
+		// some vtols fly without airspeed sensor
+		_asp_after_transition = _parameters.airspeed_trans;
```

Comment:
(obviously if you think starting with min airspeed here is a problem for the no-airspeed operation you still need to make it dependent on the mode, but minimum airspeed is the more sensible minimal start if one comes out of a time based transition, which btw you can do with the standard vtol when setting transition airspeed to 0)